### DISCUSSION TITLE
RSS button on Youtube Channels?

### DISCUSSION THREAD

**Discussion by dugite-code** [TeamNewPipe/NewPipe#10816] [Questions]
I'm not sure if it's a bug but I can't find the RSS icon on youtube channels any more. Is it just a setting I've missed? I saw this https://github.com/TeamNewPipe/NewPipe/issues/10589#issuecomment-1898626371 where it seems the OP still has the Icon but I'm not the only one who can't see it.

**Comment by opusforlife2** [comment]
Ah no, it seems to have vanished when the channel tabs feature was added. You could open a new issue asking for it to be added back.
